### DISCUSSION TITLE
Steam Integration Offline Retry

### DISCUSSION THREAD

**Discussion by Moreland-J** [FriendsOfGalaxy/galaxy-integration-steam#172] [Q&A] [answered]
Hello,

I think I'm reporting a bug but also asking a question just in case anyone has a fix for this. Whenever I try and integrate Steam games, I go through the process of username and password, Steam guard code, brief loading period and then this shows up telling me I'm Offline and giving me the option to retry.

![image](https://github.com/FriendsOfGalaxy/galaxy-integration-steam/assets/38007905/d9b5cb75-44af-4fe4-8fec-a43cd2ce1655)

Oftentimes the retry doesn't do anything unless I wait for a few minutes. Any ideas how I might fix this or if this is a larger scale bug? I've read a few forum posts that suggest this has been an issue since 2020 for other people too:

https://www.gog.com/forum/general_beta_gog_galaxy_2.0/steam_integration_not_working_after_latest_update_andor_mobile_authenticator_activation

Any help appreciated and really liked the tool when I did have it working!

**Comment by Shad0wBrok3r** [comment, marked as answer]
Is this with the default plugin or did you try with the fixed beta posted here ? https://github.com/ABaumher/galaxy-integration-steam/releases/tag/v1.0.6-beta

**Reply by Moreland-J** [reply]
Ah, it's the default plugin. Cool, I'll try the beta 👍 Thanks!

**Comment by RobinOrlic** [comment]
adding my Thumbs up, thanks! the beta fixed my issue.
I had changed the steam country and it disconnected my GOG integration it it refused the reconnect even after reinstalling GOG.
the beta fixed it! thanks!
